src/main/java/eu/bbmri_eric/negotiator/service/UserNotificationServiceImpl.java

```diff
@@ -231,7 +226,7 @@ private Notification buildNewNotification(
   }
 
   @Override
-  @Scheduled(cron = "0 0 * * * *")
+  @Scheduled(cron = "0 */1 * ? * *")
```

Comment:
Can you please add this as a configuration?

Reply:
done